library/std/src/sync/mpsc/mod.rs

```diff
@@ -1531,6 +1531,9 @@ impl<T: Send> error::Error for TrySendError<T> {
 
 #[stable(feature = "mpsc_error_conversions", since = "1.24.0")]
 impl<T> From<SendError<T>> for TrySendError<T> {
+    /// Converts a `SendError<T>` into a `TrySendError<T>`.
+    /// This conversion always returns a `TrySendError::Disconnected` containing the data in the `SendError<T>`.
```

Comment:
These need a blank `///` between the first and second lines, that is, the format is:

```
/// summary sentence
///
/// other paragraphs
```

Reply:
Also, given that the two sentences of docs here feel disconnected, I would expect these to be formatted as

```rust
/// Converts...
///
/// This conversion...
///
/// No data ...
```